public/handlers/user-auth.js

```diff
@@ -0,0 +1,127 @@
+const $signupBtn = $("#signup-btn");
+const $signupUsername = $('#signup-username');
+const $signupEmail = $('#signup-email');
+const $signupPassword = $('#signup-password');
+
+// invoked when the 'Create Account' button is clicked and sends the user information to the server for saving.
+async function handleSignUpClick(e) {
+    e.preventDefault();
+
+    console.log('going to fetch')
+    const $authModal = $('.body-modal');
+    const userData = {
+        username: $signupUsername.val().trim(),
+        email: $signupEmail.val().trim().toLowerCase(),
+        password: $signupPassword.val().trim(),
+    }
+
+    const response = await fetch('http://localhost:3001/api/user/signup', {
```

Comment:
Can we update the path to /api/user/signup instead of prepending localhost?

Reply:
Yes. you right! Initially I had it like  /api/user/signup, but since I struggled with errors (because I had no idea I would need to include preventDafault), I tried different ways to resolve it and once included preventDefault, forgot to change the URL. Just changed and tested it. It works the same with  /api/user/signup and I committed those changes